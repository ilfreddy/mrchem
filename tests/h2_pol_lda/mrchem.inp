# vim:syntax=sh:

rel_prec = 1.0e-3                 # Overall relative precision

MRA {
  min_scale = -5                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
  angstrom = true
$coords
H      0.0000 0.0000   -0.3705
H      0.0000 0.0000    0.3705
$end
}

WaveFunction {
  method = LDA                    # Wave function method (HF or DFT)
  restricted = false              # Unrestricted calculation
}

Properties {
  scf_energy = true               # Compute ground state energy
  polarizability = true           # Compute electric polarizability
}

Polarizability {
  frequency = [0.0]               # Static polarizability 
}

SCF {
  run = false                     # Use initial guess, don't optimize
  initial_guess = GTO             # Reading precomputed GTO MOs
}

Response {
  kain = 3                        # Length of KAIN iterative history
  max_iter = 10
  orbital_thrs = 1.0e-2           # Convergence threshold in orbital residual
  directions = [0,0,1]            # Compute only in z direction
}

