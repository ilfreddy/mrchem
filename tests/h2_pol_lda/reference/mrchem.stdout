


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. ab0790d0)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : 735162b

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 5
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 32.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -32.000000  -32.000000  -32.000000 ]
 upper bounds     = [   32.000000   32.000000   32.000000 ]
 total length     = [   64.000000   64.000000   64.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     H           0.00000e+00   0.00000e+00  -7.00143e-01
  2     H           0.00000e+00   0.00000e+00   7.00143e-01
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00  -2.55631e-17
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     H            1.00000e+00   1.00000e-03   1.63241e-02
  2     H            1.00000e+00   1.00000e-03   1.63241e-02
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    a    1                       9.999999974573794e-01
------------------------------------------------------------
                 Wall time: 1.56475e-01 sec 
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    b    1                       9.999999974573797e-01
------------------------------------------------------------
                 Wall time: 1.54719e-01 sec 
============================================================


============================================================
 OrbitalVector:   2 orbitals     2 occupied     2 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0     0       9.999999974574e-01    a   1    1.00000e+00
   1     0       9.999999974574e-01    b   1    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     776      1.22414e-01
 Coulomb density                       392      8.96834e-02
 Coulomb potential                       8      1.21122e-01
 XC alpha density                      392      5.77697e-02
 XC beta density                       392      4.80603e-02
 XC evaluate xcfun                    1176      8.91801e-01
 XC energy                             392      2.65610e-05
 XC potential                          784      9.28116e-03
 XC grid size                          392           594432
 XC grid change                       -384          -580608
------------------------------------------------------------
                 Wall time: 1.34218e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         1.542116410000000e-01
 Potential part                       2.586777470000000e-01
------------------------------------------------------------
                 Wall time: 4.12956e-01 sec 
============================================================


============================================================
                  Digonalizing Fock matrix
------------------------------------------------------------
 Computing Lowdin matrix                        7.07117e-04
 Diagonalizing matrix                           8.86400e-06
------------------------------------------------------------
                 Wall time: 1.66709e-02 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     776      1.18359e-01
 Coulomb density                       392      8.26444e-02
 Coulomb potential                       8      1.08847e-01
 XC alpha density                      392      4.49322e-02
 XC beta density                       392      4.38585e-02
 XC evaluate xcfun                    1176      8.83303e-01
 XC energy                             392      2.12190e-05
 XC potential                          784      9.51158e-03
 XC grid size                          392           594432
 XC grid change                       -384          -580608
------------------------------------------------------------
                 Wall time: 1.29349e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 3.13659e-01 sec 
============================================================


 XC alpha density                      392      4.08878e-02
 XC beta density                       392      4.06616e-02
 XC evaluate xcfun                    2352      2.46070e+00
 XC potential                         1960      2.32161e-02
 XC grid size                          392           594432
 XC grid change                       -384          -580608
============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                     776      1.18348e-01
 Coulomb density                       392      7.78962e-02
 Coulomb potential                       8      1.11697e-01
 XC alpha density                      392      4.07644e-02
 XC beta density                       392      4.16004e-02
 XC evaluate xcfun                    1176      8.80575e-01
 XC energy                             392      2.36650e-05
 XC potential                          784      9.19103e-03
 XC grid size                          392           594432
 XC grid change                       -384          -580608
------------------------------------------------------------
                 Wall time: 1.28206e+00 sec 
============================================================


============================================================
               Setting up Helmholtz operators
------------------------------------------------------------
 Creating operator with lambda                 -3.45284e-01
 Creating operator with lambda                 -3.45284e-01
------------------------------------------------------------
                 Wall time: 1.32060e-01 sec 
============================================================




####################### SCF cycle  1 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Perturbed Coulomb density               8      7.26770e-05
 Coulomb potential                       8      2.62540e-03
 XC perturbed alpha density              8      6.21070e-05
 XC perturbed beta density               8      5.49500e-05
------------------------------------------------------------
                 Wall time: 3.37661e-03 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
             V_0 phi_1                          7.34100e-06
             F_0 phi_1                          3.84400e-06
 (1 - rho_0) V_1 phi_0                          4.88684e-01
------------------------------------------------------------
                 Wall time: 4.94749e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.45801e+00   264    0.00000e+00     0   1.40023e+00
  1    1.45801e+00   264    0.00000e+00     0   1.41886e+00
------------------------------------------------------------
                 Wall time: 2.81917e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             2.68900e-06
 Copy orbitals                                  1.10400e-06
 Copy orbital updates                           5.71132e-03
------------------------------------------------------------
                 Wall time: 5.76124e-03 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0   1.45801e+00   1.45801e+00        392    a    1    0
  1   1.45801e+00   1.45801e+00        392    b    1    0
------------------------------------------------------------
 Total error:                            2.06193e+00    0
============================================================


============================================================
                    Value                  Update      Done 
------------------------------------------------------------
 Property     -5.523961289021099e+00    -5.52396e+00    1
============================================================


################ Wall time: 3.54132e+00 sec ################




####################### SCF cycle  2 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Perturbed Coulomb density             392      8.97007e-02
 Coulomb potential                     200      8.10439e-01
 XC perturbed alpha density            392      4.51475e-02
 XC perturbed beta density             392      4.56964e-02
------------------------------------------------------------
                 Wall time: 9.92192e-01 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
             V_0 phi_1                          3.04372e-01
             F_0 phi_1                          4.59200e-06
 (1 - rho_0) V_1 phi_0                          4.78091e-01
------------------------------------------------------------
                 Wall time: 7.98345e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.77189e+00   392    0.00000e+00     0   2.68513e+00
  1    1.77189e+00   392    0.00000e+00     0   2.69751e+00
------------------------------------------------------------
                 Wall time: 5.38273e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.61400e-06
 Setup linear system                            1.45597e-02
 Solve linear system                            2.62300e-06
 Expand solution                                2.01204e-02
------------------------------------------------------------
                 Wall time: 3.47666e-02 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0   1.85794e+00   3.17142e-01        392    a    1    0
  1   1.85794e+00   3.17142e-01        392    b    1    0
------------------------------------------------------------
 Total error:                            4.48507e-01    0
============================================================


============================================================
                    Value                  Update      Done 
------------------------------------------------------------
 Property     -7.124922222968479e+00    -1.60096e+00    1
============================================================


################ Wall time: 7.42552e+00 sec ################




####################### SCF cycle  3 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Perturbed Coulomb density             392      1.10025e-01
 Coulomb potential                     200      8.68097e-01
 XC perturbed alpha density            392      5.40951e-02
 XC perturbed beta density             392      5.44265e-02
------------------------------------------------------------
                 Wall time: 1.08787e+00 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
             V_0 phi_1                          3.05532e-01
             F_0 phi_1                          4.32900e-06
 (1 - rho_0) V_1 phi_0                          4.80602e-01
------------------------------------------------------------
                 Wall time: 8.01765e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.86499e+00   392    0.00000e+00     0   2.81029e+00
  1    1.86499e+00   392    0.00000e+00     0   2.79285e+00
------------------------------------------------------------
                 Wall time: 5.60324e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.31009e-03
 Setup linear system                            4.57381e-02
 Solve linear system                            3.99500e-06
 Expand solution                                3.59718e-02
------------------------------------------------------------
                 Wall time: 8.31107e-02 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0   1.87067e+00   2.22533e-02        456    a    1    0
  1   1.87067e+00   2.22533e-02        456    b    1    0
------------------------------------------------------------
 Total error:                            3.14709e-02    0
============================================================


============================================================
                    Value                  Update      Done 
------------------------------------------------------------
 Property     -7.211762903095228e+00    -8.68407e-02    1
============================================================


################ Wall time: 7.81525e+00 sec ################




####################### SCF cycle  4 #######################


============================================================
 Current precision                              1.00000e-03
------------------------------------------------------------
 Orbital threshold                              1.00000e-02
 Property threshold                            -1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Perturbed Coulomb density             392      1.10127e-01
 Coulomb potential                     200      8.93167e-01
 XC perturbed alpha density            392      5.35634e-02
 XC perturbed beta density             392      5.30486e-02
------------------------------------------------------------
                 Wall time: 1.11113e+00 sec 
============================================================


============================================================
               Setting up Helmholtz arguments
------------------------------------------------------------
             V_0 phi_1                          3.54481e-01
             F_0 phi_1                          4.51000e-06
 (1 - rho_0) V_1 phi_0                          4.80512e-01
------------------------------------------------------------
                 Wall time: 8.52625e-01 sec 
============================================================


============================================================
                Applying Helmholtz operators
------------------------------------------------------------
 Orb    RealNorm   Nodes     ImagNorm   Nodes     Timing
------------------------------------------------------------
  0    1.87087e+00   392    0.00000e+00     0   2.82208e+00
  1    1.87087e+00   392    0.00000e+00     0   2.82124e+00
------------------------------------------------------------
                 Wall time: 5.64341e+00 sec 
============================================================


============================================================
               Iterative subspace accelerator
------------------------------------------------------------
 Push back orbitals                             1.31211e-03
 Setup linear system                            5.92664e-02
 Solve linear system                            5.82500e-06
 Expand solution                                3.74526e-02
------------------------------------------------------------
                 Wall time: 9.81329e-02 sec 
============================================================


============================================================
                          Orbitals
------------------------------------------------------------
 Orb    F(i,i)        Error         nNodes  Spin  Occ  Done 
------------------------------------------------------------
  0   1.87166e+00   3.95606e-03        456    a    1    1
  1   1.87166e+00   3.95606e-03        456    b    1    1
------------------------------------------------------------
 Total error:                            5.59471e-03    1
============================================================


============================================================
                    Value                  Update      Done 
------------------------------------------------------------
 Property     -7.222008290208747e+00    -1.02454e-02    1
============================================================


################ Wall time: 7.94360e+00 sec ################


============================================================
                      Convergence rate
------------------------------------------------------------
Iter    OrbError       Property                   Update  
------------------------------------------------------------
  1    2.06193e+00    -5.523961289021099e+00   -5.52396e+00
  2    4.48507e-01    -7.124922222968479e+00   -1.60096e+00
  3    3.14709e-02    -7.211762903095228e+00   -8.68407e-02
  4    5.59471e-03    -7.222008290208747e+00   -1.02454e-02
------------------------------------------------------------
                      SCF converged!!!                      
============================================================


============================================================
                 Calculating polarizability
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 6.28317e-01 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-3.45284e-01  0.00000e+00
 0.00000e+00 -3.45284e-01
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    a    1                      -3.452838805081155e-01
    1    b    1                      -3.452838805081154e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     H           0.00000e+00   0.00000e+00  -7.00143e-01
  2     H           0.00000e+00   0.00000e+00   7.00143e-01
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00  -2.55631e-17
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -6.905677610162310e-01
 Kinetic energy:                      1.073485968172088e+00
 E-N energy:                         -3.566631913545357e+00
 Coulomb energy:                      1.328364211798184e+00
 Exchange energy:                     0.000000000000000e+00
 X-C energy:                         -6.538705241579348e-01
 El. External field energy:           0.000000000000000e+00
 Nuc. External field energy:          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -1.818652257733020e+00
 Nuclear energy:                      7.141393372477786e-01
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -1.104512920485242e+00
                    (kJ/mol)         -2.899898253019092e+03
                    (kcal/mol)       -6.930923162373331e+02
                    (eV)             -3.005532637202394e+01
                                                            
============================================================
                                                            

                                                            
============================================================
                   Polarizability tensor                    
------------------------------------------------------------
                                                            
 Frequency:       (au)                     0.0000000000e+00
                                                            
 Isotropic average(au)                     2.4073360967e+00
 Isotropic average(SI)       TO BE FIXED                    
                                                            
-------------------------- Tensor --------------------------
                                                            
   0.0000000000e+00    0.0000000000e+00    3.3571225380e-15
   0.0000000000e+00    0.0000000000e+00    1.8876564578e-15
   3.3571225380e-15    0.0000000000e+00    7.2220082902e+00
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 3.511321e+01                ***
***                                                      ***
************************************************************


